Fix #253: Use AdcStart() before each ReadAnalog value

dhineshkumarmcci opened the pull request:
Added `AdcStart()` call before each `AdcGetValue()` to read the Channel value. With long time run, this fixed the `?CatenaStm32L0::ReadAnalog(3): CatenaStm32L0_ReadAnalog() failed (5)`, as reported in issue #253

terrillmoore commented:
@chwon64, can you please check?

terrillmoore commented:
I reverted this. **We are required to update version numbers and documentation.** This is not a new requirement. Please bear this in mind and update. Since this is not a release, the last (x.y.z.last) field of the version number must be advanced (but do not change x.y.z); and you **must** update the release history in `README.md`. If you review that, you'll see that for pre-releases I say:

> HEAD contains the following changes.
> - Fix #253 by calling AdcStart() before each read.

Please bear this in mind, and check all other pending pull requests.

dhineshkumarmcci commented:
@terrillmoore I have updated the version number and updated release history in `README.md`. Also I have submitted a new pull request #256 for the same.
